tests/unit/chrome_client_test.ts

```diff
@@ -54,6 +55,65 @@ Rhum.testPlan("tests/unit/chrome_client_test.ts", () => {
         // Unable to test properly, as windows doesnt like 0.0.0.0 or localhost, so the only choice is 127.0.0.1 but this is already the default
       },
     );
+    Rhum.testCase(
+      "Uses the binaryPath when passed in to the parameters",
+      async () => {
+        async function getChromePath(): Promise<string> {
```

Comment:
To DRY up the code, this looks identical to the method in the chrome source code file? If so we should probably reuse that (could abstract the logic outside of the class?) ie:

```ts
export async function getChromePath() {
  // ... all logic here
}

class Chrome {
  ...
  private static async getChromePath() { 
    return getChromePath()
  }
}
```

Reply:
This feels good, will abstract it out, for both firefox and chrome

Reply:
I have a question, if we are abstracting it out, do we still need a class method to call it?. Can't we just call the actual logic method?

Reply:
I guess we don't really, I prefer your approach better :) There wasn't realy a need to have a class method, resolving :)